src/models/posts.ts

```diff
@@ -512,7 +512,10 @@ const posts = (sequelize: Sequelize) => {
               WHERE ts @@ to_tsquery('english', :query)
               ORDER BY "createdAt" ${order} LIMIT :limit
               OFFSET :offset`,
-        { type: QueryTypes.SELECT, replacements: { limit, offset, query } }
+        {
+          type: QueryTypes.SELECT,
+          replacements: { limit, offset, query: query.replace(/\||,/g, '').replace(/ /g, ' | ') },
+        }
```

Comment:
Shouldn't it rather be something like
```suggestion
        {
          type: QueryTypes.SELECT,
          replacements: {
            limit,
            offset,
            query: query.replace(/\||,/g, ' ').replace(/\s\s+/g, ' ').replace(/ /g, ' | '),
          },
```
Where the regex:
1. replaces `,` or `|` chars by `' '` (single whitespace)  
2. Replaces multiple whitespaces by single whitespace
3. Replaces single whitespace by ` | `

Suggested fix and unit tests: https://github.com/zkitter/zkitterd/pull/57